code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -42,6 +48,17 @@
 		return ..()
 
 
+/mob/living/simple_animal/hostile/illusion/AttackingTarget()
+	..()
+	if(istype(target, /mob/living) && prob(multiply_chance))
+		var/mob/living/L = target
+		if(L.stat == DEAD)
+			return
+		var/mob/living/simple_animal/hostile/illusion/M = new(loc)
+		M.faction = faction
```

Comment:
Speed isn't an issue on anything that isn't atmos. And this shouldn't make them attack each other since they all still have their greytide faction.